core/src/main/resources/db/migration/V10__add_update_spec_install_pos.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE UpdateRequest
```

Comment:
- `UpdateRequest`s aren't indexed by vehicle, `UpdateSpec`s are
- Requirement: installation queue _per vehicle_ (which makes sense: a package install failing on one vehicle shouldn't block the installation queue of others)
- Terminology: "installation queue" (for a vehicle) are those pending `UpdateSpec`s for that vehicle.